packages/eui/src/components/datagrid/body/header/data_grid_header_cell_wrapper.styles.ts

```diff
@@ -48,6 +47,14 @@ export const euiDataGridHeaderCellWrapperStyles = (
         ${hoverStyles}
       }
 
+      ${outlineSelectors.showActions} {
+        gap: ${euiTheme.size.xxs};
+
+        & > [data-focus-lock-disabled] {
+          gap: ${euiTheme.size.xxs};
+        }
```

Comment:
[optional] we can DRY out this CSS by comining selectors like so:

```suggestion
        &,
        & > [data-focus-lock-disabled] {
          gap: ${euiTheme.size.xxs};
        }
```

Reply:
Actually sorry, I'm not quite clear on why we moved this out of the `_sharedFlexCss`. Couldn't we just have added the `${outlineSelectors.showActions} { }` wrapper there?